src/main/java/core/basesyntax/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    private static final String HEADER = "fruit,quantity";
+
+    @Override
+    public void writeFile(Map<String, String> conclusionData, String separator, String fileName) {
```

Comment:
```suggestion
    public void writeToFile(Map<String, String> conclusionData, String separator, String fileName) {
```

Reply:
do you need to pass separator? I guess that for this particular implementation it always be a comma. We can make constant separator in this class

Reply:
DONE